src/util/DeepLinkParser.js

```diff
@@ -21,6 +21,10 @@ export function parseDeepLink(uri: string, opts: { aztecoApiKey?: string } = {})
 
   const url = new URL(uri)
 
+  if (url.protocol === 'dev:') {
```

Comment:
Please block this feature behind the development build flag so it won't work on live builds